packages/@orchidui-vue/src/Builder/DataTable/OcDataTable.vue

```diff
@@ -79,11 +79,13 @@ const queries = ref(defaultQuery ? defaultQuery.split(",") : []);
 const isSearchExpanded = ref(false);
 
 const customPerPageOptions = computed(() => {
-  return props.options?.perPageOptions?.map(perPage => ({
-    label: `${perPage}`,
-    value: perPage,
-  })) ?? null;
-})
+  return (
+    props.options?.perPageOptions?.map((perPage) => ({
+      label: `${perPage}`,
+      value: perPage,
+    })) ?? null
+  );
+});
```

Comment:
This can be shortened to:
```
const customPerPageOptions = computed(() =>
    props.options?.perPageOptions?.map((perPage) => ({
      label: `${perPage}`,
      value: perPage,
    }) ?? null);
    );
```